src/styles/utils/reset.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
only `box-sizing: border-box;` is allowed to use at `*` selector

Reply:
:arrow_up: :+1: